src/test/test_addr.c

```diff
@@ -1750,6 +1750,49 @@ test_addr_is_valid(void *arg)
   ;
 }
 
+#define TEST_ADDR_IS_NULL(a, rv) STMT_BEGIN               \
```

Comment:
Thanks for these new unit tests.

Let's keep them, even if we don't change tor_addr_is_null().